terraform/gcp/.env.example

```diff
@@ -0,0 +1,89 @@
+# terraform variables
```

Comment:
I am wondering if we could have a single config file for both cloud providers and overwrite the env vars which really differ, as some vars are identical for both of them, and also it'd be harder to maintain two env files.

Reply:
Done. One _.env_ for both AWS and GCP